### PR TITLE
fix: type error

### PR DESCRIPTION
**Pull request by mikerodonnell89**
fixes none

Type error, possibly related to new dependencies in #9391 but this fixes it ahead of time

### CODE REVIEW

File: libs/cdk/src/lib/utils/directives/focusable-item/focusable-item.directive.ts [hunk 1]
```diff
@@ -84,7 +84,7 @@ export class FocusableItemDirective implements HasElementRef {
     private _tabbable = true;
 
     /** @hidden */
-    private _timerId: number | null;
+    private _timerId: any | null;
```
Comment (N1XUS) [inline libs/cdk/src/lib/utils/directives/focusable-item/focusable-item.directive.ts:87]: You can use https://www.typescriptlang.org/docs/handbook/utility-types.html#returntypetype:
```
private _timerId: ReturnType<typeof setTimeout> | null;
```

**Review by N1XUS — APPROVED** [review]

File: libs/cdk/src/lib/utils/directives/focusable-item/focusable-item.directive.ts [hunk 2]
```diff
@@ -84,7 +84,7 @@ export class FocusableItemDirective implements HasElementRef {
     private _tabbable = true;
 
     /** @hidden */
-    private _timerId: number | null;
+    private _timerId: ReturnType<typeof setTimeout> | null;
```
Comment (g-cheishvili) [inline libs/cdk/src/lib/utils/directives/focusable-item/focusable-item.directive.ts:87]:
```suggestion
    private _timerId: ReturnType<typeof global.setTimeout> | null;
```
This should fix it. Basically ts confuses node and web environments

File: libs/cdk/src/lib/utils/directives/focusable-item/focusable-item.directive.ts (hunk 1 above)
Comment (mikerodonnell89) [reply libs/cdk/src/lib/utils/directives/focusable-item/focusable-item.directive.ts]: thanks!

File: libs/cdk/src/lib/utils/directives/focusable-item/focusable-item.directive.ts (hunk 2 above)
Comment (mikerodonnell89) [reply libs/cdk/src/lib/utils/directives/focusable-item/focusable-item.directive.ts]: sounds good

**Pull request merged**